app/test_main.py

```diff
@@ -1,3 +1,25 @@
 from app.main import get_human_age
 
-# write your code here
+def test_should_return_zero_if_zero_input():
+    assert get_human_age(0, 0) == [0, 0]
+
+def test_should_return_zero_if_less_than_cat_year():
+    assert get_human_age(14, 14) == [0, 0]
+
+def test_minimum_non_zero_result():
+    assert get_human_age(15, 15) == [1, 1]
+
+def test_minimum_non_zero_result():
+    assert get_human_age(23, 23) == [1, 1]
+
+def test_should_return_zero_if_less_than_cat_year():
+    get_human_age(24, 24) == [2, 2]
+
+def test_max_two_years_result():
+    get_human_age(27, 27) == [2, 2]
```

Comment:
The test `test_max_two_years_result` is missing an assertion. It should use `assert` to verify the expected output.